.github/workflows/ci.yaml

```diff
@@ -21,6 +21,8 @@ jobs:
     continue-on-error: ${{ matrix.ruby == 'ruby-head' }}
     steps:
       - uses: actions/checkout@v4
+      - name: Install ripgrep
```

Comment:
In a future PR, we probably want to adopt the `dev` way here and have a key users can use to verify a requirement has been met and not need to run this command every time.